modules/oc-mirror-disk-to-mirror.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-mirroring-disconnected.adoc
+
+[id="oc-mirror-disk-to-mirror_{context}"]
+= Mirroring from disk to a mirror in a disconnected environment
+
+// need "in a disconnected environment"?
+
+// TODO: intro
+
+////
+Note: When using oc-mirror lifecycle operations, the above commands are continually reused to create and publish imageset updates as needed. The imageset-config can also be modified between application runs as needed. This promotes oc-mirror usage in workflow automation or cron jobs.
+
+TODO: what exactly does this mean. says that the above commands are continually reused (does this mean automatically, or that users can continually manually reuse them as often as wanted)? Or have to set up their own workflow automation/cron jobs?
+
+TODO: Figure out more on use cases for updating the image set config file before runs - and document that somewhere
+
+////
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`) in the disconnected environment.
+* You have installed the `oc-mirror` CLI plug-in in the disconnected environment.
+* You have created the image set configuration file.
+* You have generated the image set file by using the `oc mirror` command.
+* You have transferred the image set file and the image set configuration file to the disconnected environment.
+// TODO: Confirm prereq about not needing a cluster, but need pull secret misc
+
+// TODO: confirm that they needed to transfer the same image set config file to the disconnected env
+
+.Procedure
+
+* Run the `oc mirror` command to process the image set file on disk and mirror the contents to a target mirror registry:
++
+[source,terminal]
+----
+$ oc mirror --from=./mirror_seq1_000000.tar \ <1>
+  docker://registry.example:5000/             <2>
+----
+<1> Pass in the image set .tar file to mirror. This procedure assumes that it is named `mirror_seq1_000000.tar`. You can also pass in a directory that contains the image set .tar file.
+<2> Specify the registry to mirror the image set file to. The registry must start with `docker://`.
++
+// TODO: for <1>: Can there be more than one .tar file in the directory?
+// TODO: for <2>: Confirm that the docker:// start is an actual requirement.
+
+// TODO: mention here that it updates the mirror and then also generates the manifests.
+
+.Verification
+
+* TODO
+
+// TODO: Verification steps - check for manifests. What is the output directory? Wherever you ran the oc mirror command from? Does the oc mirror command create taht directory if it doesn't exist, and create the results-xxx... subfolder. Does the end of the command say which results-xxxx folder it is, just in case there are multiple?
```

Comment:
@afflom @jpower432 I need a bit of help with what happens after you run this disk2mirror command. If either of you has any example output that you could share, that would help. Also:

* Does it just create the `oc-mirror-workspace` directory in whatever directory the user is in when they run `oc mirror`?
* Does the end of the command give output that shows which results-xxxx folder contains the manifests from this run? (just in case there are multiple)